crates/client/rpc-core/src/utils.rs

```diff
@@ -293,8 +321,10 @@ pub fn casm_contract_class_to_compiled_class(casm_contract_class: &CasmContractC
         compiler_version: casm_contract_class.compiler_version.clone(),
         bytecode: casm_contract_class.bytecode.iter().map(|x| biguint_to_field_element(&x.value)).collect(),
         entry_points_by_type: casm_entry_points_to_compiled_entry_points(&casm_contract_class.entry_points_by_type),
-        hints: vec![],        // not needed to get class hash so ignoring this
-        pythonic_hints: None, // not needed to get class hash so ignoring this
+        // TODO: fill those
```

Comment:
Truth is I'm not even sure we really need to fill them. 
Maybe they should be stored in the client. Because we don't need them for the execution, so no reason to have them in the runtime. Wdyt?

Reply:
right now its being used in a generic function so i thouht it would make sense to fill it, but otherwise let's add back the comment that its not needed and currently only used for class hash calculation. we can also not it make it public perhaps so someone doesn't use the function incorrectly.

Reply:
All good points

Reply:
But I think we should go for solution where we pass an extra bool arg `fill_extra_field` or something.
Because we will need those for the RPC get_class_at, we are supposed to return a struct with those field filled.